mapsforge-core/src/main/java/org/mapsforge/core/util/Parameters.java

```diff
@@ -49,8 +50,12 @@ public enum SymbolScaling {ALL, POI}
 
     /**
      * Use AwtLuminanceShadingComposite or AlphaComposite.
+     * (2024: Deprecated, should not be used)
      */
-    public static boolean LUMINANCE_COMPOSITE = true;
+    public static final boolean LUMINANCE_COMPOSITE = false;
```

Comment:
@Sublimis Parameters cannot be final. Either keep it non final or remove the `LUMINANCE_COMPOSITE`.